packages/core/src/__mocks__/data-source.ts

```diff
@@ -19,24 +18,34 @@ export const createMockSiteWiseDataSource = (
   } = { dataStreams: [], onRequestData: () => {} }
 ): DataSource<SiteWiseDataStreamQuery> => ({
   name: 'site-wise',
-  initiateRequest: jest.fn(({ query, request, onSuccess = () => {} }: DataSourceRequest<SiteWiseDataStreamQuery>, requestInformations: RequestInformationAndRange[]) => {
-    query.assets.forEach(({ assetId, properties }) =>
-      properties.forEach(({ propertyId }) => {
-        const correspondingRequestInfo = requestInformations.find(({ id }) => `${ assetId }---${propertyId}` === id);
-        if (correspondingRequestInfo) {
-          onRequestData({ assetId, propertyId, request });
-          onSuccess(dataStreams, 'fetchFromStartToEnd', correspondingRequestInfo.start, correspondingRequestInfo.end);
-        }
-      })
-    );
-  }),
+  initiateRequest: jest.fn(
+    (
+      { query, request, onSuccess = () => {} }: DataSourceRequest<SiteWiseDataStreamQuery>,
+      requestInformations: RequestInformationAndRange[]
+    ) => {
+      query.assets.forEach(({ assetId, properties }) =>
+        properties.forEach(({ propertyId }) => {
+          const correspondingRequestInfo = requestInformations.find(({ id }) => `${assetId}---${propertyId}` === id);
```

Comment:
well want to use use the `toId` method (though i see this was existing code)

Reply:
I'm going to leave this for now, this mock is in core, and `toId` is in source-iotsitewise so it'll be a circular dependency to import it. In the future we'll probably want to move this mock out of core.